### PR TITLE
Settings page: Update "Track logged-in users" option texts for clarity

### PR DESCRIPTION
**Pull request by acicovic**
## Description
Our settings page was not entirely clear on what the "Track logged-in users" option does. This PR renames the option and attempts to make things clearer in the option's help text, as well as in other parts of the code. 

## Motivation and Context
Make our setting descriptions more accurate, and our code easier to understand.

## How Has This Been Tested?
Tests pass.

### CODE REVIEW

**Comment by acicovic** [comment]
We have an alternative text proposition by @chriszarate:

>Choose whether logged-in visitors should be tracked. You will no longer see the [Parse.ly](http://parse.ly/) tracking code on your site if you browse while logged-in. Note: For WordPress multisite, a user must be logged-in to the current site to be considered logged-in.

**Comment by acicovic** [comment]
Updated to suggested text.

File: src/UI/class-settings-page.php [hunk 1]
```diff
@@ -401,7 +401,7 @@ private function initialize_basic_section(): void {
 		);
 
 		// Track logged-in users.
-		$h = __( 'By default, the plugin will track the activity of users that are logged into this site. You can change this setting to only track the activity of anonymous visitors. Note: You will no longer see the Parse.ly tracking code on your site if you browse while logged in.', 'wp-parsely' );
+		$h = __( 'Choose whether logged-in visitors should be tracked. You will no longer see the Parse.ly tracking code on your site if you browse while logged-in. Note: For WordPress multisite, a user must be logged-in to the current site to be considered logged-in.', 'wp-parsely' );
```
Comment (GaryJones) [inline src/UI/class-settings-page.php]: Can we make the bit about WP multisite conditionally shown if this is actually a multisite install (`is_multisite()`)? No point in telling single-site install users something that doesn't affect them.

Comment (acicovic) [reply src/UI/class-settings-page.php]: Done!

**Review by GaryJones — APPROVED** [review]
✅

**Pull request merged**